### DISCUSSION TITLE
Enhance the functionality of the second canvas view

### DISCUSSION THREAD

**Discussion by wyattbiker** [Orama-Interactive/Pixelorama#581] [Ideas]
Right Panel drag to the left shows copy of image. Not sure if it is intentional.

**Comment by OverloadedOrama** [comment]
This is intentional. It's meant to act as a split view mode that lets you see your canvas twice. It's useful for when you want to look at different positions or zoom levels of the image at the same time. In the future we could expand it to let users be able to view different frames too.

**Comment by Schweini07** [comment]
> In the future we could expand it to let users be able to view different frames too.
Maybe rename the issue, so it's more fitting?!
